packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -262,6 +264,17 @@ describe('@api RPC Server Acceptance Tests', function () {
                     expect(logsWithTopic[i].topics[0]).to.be.equal(topic);
                 }
             });
+
+            it('should be able to return more than 2 logs with limit of 2 logs per request', async () => {
+                //for the purpose of the test, we are settings limit to 2, and fetching all. 
+                const blocksBehindLatest = Number(await relay.call('eth_blockNumber', [], requestId)) - 50;
+                const logs = await relay.call('eth_getLogs', [{
+                    'fromBlock': blocksBehindLatest,
+                    'toBlock': 'latest',
+                    'limitParams': { limit: 2, order: 'desc'}
```

Comment:
As I mentioned in the previous comment, in my opinion we should not change the `eth_getLogs` by adding custom properties to the Filter object

Reply:
This is optional parameter and it's needed for testing. Otherwise we have to log more than 100 logs per test. This way, I'm limiting it to 2 (for example) and test the functionality easy.

Reply:
This seems as a wrong approach, imo. There should be another way to do this, like setting the limit as an env var and set the env var before the test case.

Reply:
Done. Added `.env` variable